packages/design-system/src/SegmentedControl/SegmentedControl.styles.tsx

```diff
@@ -0,0 +1,66 @@
+import styled from "styled-components";
+
+export const StyledSegmentedControl = styled.div<{
+  isDisabled?: boolean;
+}>`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: var(--ads-v2-spaces-2);
+  background-color: var(--ads-v2-color-bg-muted);
+  color: var(--ads-v2-color-fg);
+  padding: var(--ads-v2-spaces-1);
+  border-radius: var(--ads-v2-border-radius);
+`;
+
+export const ControlContainer = styled.div`
+  display: flex;
+  align-items: center;
+  position: relative;
+  border-radius: var(--ads-v2-border-radius);
+  background-color: transparent;
+  cursor: pointer;
+  transition: box-shadow 600ms ease;
+
+  &[data-disabled="true"] {
+    opacity: var(--ads-v2-opacity-disabled);
+    cursor: not-allowed !important;
+  }
+
+  &[data-selected="true"] {
+    background-color: var(--ads-v2-color-bg);
+    box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.12);
+  }
+
+  &:not(:last-child):not([data-selected="true"]):after {
```

Comment:
And is this to mask the separator on the right? Can you add a comment here ?

Reply:
no this is not masking this is a straight forward selector